databricks_cli/dbfs/api.py

```diff
@@ -114,15 +114,12 @@ def get_status(self, dbfs_path, headers=None):
         return FileInfo.from_json(json)
 
     def put_file(self, src_path, dbfs_path, overwrite, headers=None):
-        handle = self.client.create(dbfs_path.absolute_path, overwrite, headers=headers)['handle']
-        with open(src_path, 'rb') as local_file:
-            while True:
-                contents = local_file.read(BUFFER_SIZE_BYTES)
-                if len(contents) == 0:
-                    break
-                # add_block should not take a bytes object.
-                self.client.add_block(handle, b64encode(contents).decode(), headers=headers)
-            self.client.close(handle, headers=headers)
+        self.client.put(dbfs_path.absolute_path, src_path=src_path,
+                        overwrite=overwrite, headers=headers)
+
+    def put_content(self, content, dbfs_path, overwrite, headers=None):
```

Comment:
A comment here about the need and two different functions would be useful for future context.